packages/sw-background-sync-queue/demo/sw.js

```diff
@@ -26,6 +24,7 @@ let bgQueue = new goog.backgroundSyncQueue.BackgroundSyncQueue({callbacks:
 		},
 		onRetryFailure: (hash) => {},
 	},
+	dbName: 'queues',
```

Comment:
I was just testing if the new constructor working.
Its totally optional. 
Will it be better if I remove it?